examples/api/src/app/api/farcaster/channels/route.ts

```diff
@@ -1,25 +1,41 @@
-import { NextResponse, NextRequest } from "next/server";
-import channels from "./channels.json";
-
-// Update Channels from: https://github.com/neynarxyz/farcaster-channels/blob/main/warpcast.json
-
-const channelsWithHome = [
-  {
-    name: "Home",
-    parent_url: null,
-    image: "https://warpcast.com/~/channel-images/home.png",
-    channel_id: "home",
-  },
-  ...channels.sort((a, b) => (a.name < b.name ? -1 : 1)),
-];
+import { NextRequest, NextResponse } from "next/server";
 
 export async function GET(request: NextRequest) {
   const { searchParams } = new URL(request.url);
   const q = searchParams.get("q");
 
+  const channelsRes = await fetch(
+    "https://api.neynar.com/v2/farcaster/channel/list",
```

Comment:
how do you feel about this re opencast? I know you wanted an external api-less solution for opencast

Reply:
since channels are basically warpcast-only at this stage there is no other option for the time being so this is the recommended method (see this thread with dan and rish https://warpcast.com/dwr.eth/0xaa065936)